charts/kyverno/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+Thank you for installing {{ .Chart.Name }} 😀
+
+Your release is named {{ .Release.Name }}.
+
+Kyverno chart is installed in "default" Pod Security Standards have been automatically deployed and in audit mode.
+
+Please visit this https://kyverno.io/policies/ to know about policies.
```

Comment:
"Visit https://kyverno.io/policies/ to find more sample policies."

Reply:
Done